docs/source/openshift/index.adoc

```diff
@@ -0,0 +1,13 @@
+[[interacting-with-openshift]]
+= Interacting with OpenShift
+:icons:
+
+Minishift creates a virtual machine (VM) and provisions a local, single-node OpenShift
```

Comment:
On second thought. None of the index pages has a second level heading and "Overview" section. They are also not including a TOC. So I think this is actually correct, at least looking at the current usage.